samples/cellular/modem_shell/README.rst

```diff
@@ -1518,6 +1518,15 @@ For example:
 
          west build -p -b nrf9160dk_nrf9160_ns -- -DOVERLAY_CONFIG="overlay-modem-trace-flash.conf;overlay-memfault.conf" -DDTC_OVERLAY_FILE=nrf9160dk_ext_flash.overlay
 
+Enabling TFM logging with modem traces
```

Comment:
TF-M logging is already moved to UART0 in `modem_shell` prj.conf, so this is not necessary. We want to always have the TF-M logging visible in the app UART.

Reply:
Ahh, ok! Then I'll remove the text! Sorry, I did not check that.

Reply:
Also here TM-F, if this section is necessary.